extensions/mssql/src/objectManagement/constants.ts

```diff
@@ -28,6 +28,7 @@ export const AlterApplicationRoleDocUrl = 'https://learn.microsoft.com/sql/t-sql
 export const CreateDatabaseRoleDocUrl = 'https://learn.microsoft.com/sql/t-sql/statements/create-role-transact-sql';
 export const AlterDatabaseRoleDocUrl = 'https://learn.microsoft.com/sql/t-sql/statements/alter-role-transact-sql';
 export const CreateDatabaseDocUrl = 'https://learn.microsoft.com/sql/t-sql/statements/create-database-transact-sql';
+export const ViewServerPropertiesDocUrl = 'https://learn.microsoft.com/en-us/sql/t-sql/functions/serverproperty-transact-sql?view=sql-server-ver16';
```

Comment:
By using en-us url, are we restricting the users to redirect to English only? @Charles-Gagnon @erinstellato-ms ?

Reply:
oh good catch! the other links redirect to the translated version that the user can choose in the Microsoft learn page. 
<img width="203" alt="image" src="https://github.com/microsoft/azuredatastudio/assets/34872381/9d596555-6582-450d-9a88-e4dbb8e62b24">